lib/core/resolvers/GitResolver.js

```diff
@@ -26,6 +26,9 @@ function GitResolver(decEndpoint, config, logger) {
     // anyway
     mkdirp.sync(config.storage.empty);
     process.env.GIT_TEMPLATE_DIR = config.storage.empty;
+    if (!config.strictSsl) {
+        process.env.GIT_SSL_NO_VERIFY = 'true';
+    }
```

Comment:
GIT_TEMPLATE_DIR is set every time (if config.storage.empty is false or true), and GIT_SSL_NO_VERIFY only if config.strictSsl is false. You don't handle the case when config.strictSsl is true.